qa/zcash/performance-measurements.sh

```diff
@@ -203,6 +203,9 @@ case "$1" in
             sendtoaddress)
                 zcash_rpc zcbenchmark sendtoaddress 10 "${@:4}"
                 ;;
+            loadwallet)
+                zcash_rpc zcbenchmark loadwallet 10 "${@:4}"
```

Comment:
Non-blocking: this isn't technically necessary, as the benchmark doesn't itself take any arguments.